include/libswiftnav/track.h

```diff
@@ -231,9 +255,18 @@ void lock_detect_init(lock_detect_t *l, float k1, float k2, u16 lp, u16 lo);
 void lock_detect_reinit(lock_detect_t *l, float k1, float k2, u16 lp, u16 lo);
 void lock_detect_update(lock_detect_t *l, float I, float Q, float DT);
 
-void cn0_est_init(cn0_est_state_t *s, float bw, float cn0_0,
-                  float cutoff_freq, float loop_freq);
-float cn0_est(cn0_est_state_t *s, float I, float Q);
+void cn0_est_bl_init(cn0_est_state_t *s, float bw, float cn0_0, float f_s, float f_i);
+float cn0_est_bl_update(cn0_est_state_t *s, float I, float Q);
+void cn0_est_snv_init(cn0_est_state_t *s, float bw, float cn0_0, float f_s, float f_i);
+float cn0_est_snv_update(cn0_est_state_t *s, float I, float Q);
+
+void lp1_filter_init(lp1_filter_t *f, float initial,
+                     float cutoff_freq, float loop_freq);
+float lp1_filter_update(lp1_filter_t *f, float sample);
+
+void bw2_filter_init(bw2_filter_t *f, float initial,
+                     float cutoff_freq, float loop_freq);
+float bw2_filter_update(bw2_filter_t *f, float value);
 
```

Comment:
Filter updates take different variable names, "sample vs. value". Could it be the same name for both?